src/main/resources/xslt-files/EMD_doi_datacite_v4.1.xsl

```diff
@@ -779,13 +779,50 @@
     <!-- emd:rights/dct:accessRights to datacite rights -->
     <!-- ==================================================== -->
     <xsl:template match="emd:rights">
-        <xsl:apply-templates select="dcterms:accessRights"/>
-
+        <xsl:variable name="access-rights" select="dcterms:accessRights"/>
         <xsl:variable name="licenses" select="dcterms:license"/>
+        <xsl:variable name="doi" select="//emd:identifier/dc:identifier[@eas:scheme='DOI']/text()"/>
+        <xsl:variable name="origin-doi" select="if (starts-with($doi, '10.17026/dans-')) then 'DANS' else 'OTHER'"/>
```

Comment:
```suggestion
        <xsl:variable name="origin-doi" select="if (starts-with($doi, '10.17026/')) then 'DANS' else 'OTHER'"/>
```

Reply:
the prefix is DANS', it is just a pattern we start with `dans-` now, that might change

Reply:
Done